lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -3,34 +3,36 @@
     from django.utils.translation import ugettext as _
 %>
 <%namespace name='static' file='../static_content.html'/>
-<div class="message message-status is-shown credit-message">
-    <div class="xseries-action">
-        <div class="message-copy xseries-msg">
-            <p>
-                <b class="message-copy-bold">${_("{category} Program: Interested in more courses in this subject?").format(category=course_program_info['display_category'])}</b>
-            </p>
-            <p class="message-copy">
-                ${_("This course is 1 of {course_count} courses in the {link_start}{program_display_name}{link_end} {program_category}.").format(
-                    course_count=course_program_info['course_count'],
-                    link_start='<a href="{}">'.format(course_program_info['program_marketing_url']),
-                    link_end='</a>',
-                    program_display_name=course_program_info['display_name'],
-                    program_category=course_program_info['display_category'],
-                )}
-            </p>
+%for program_data in course_program_info.get('course_program_list', {}):
+    <div class="message message-status is-shown credit-message">
+        <div class="xseries-action">
+            <div class="message-copy xseries-msg">
+                <p>
+                    <b class="message-copy-bold">${_("{category} Program: Interested in more courses in this subject?").format(category=course_program_info.get('display_category', ''))}</b>
```

Comment:
I know this was already implemented and the PR is updating to implement multiple when present, but why is there a ```<b>``` with the ```message-copy-bold``` class within the ```<p>``` when there are no more elements present? Could the class be applied to the ```<p>``` and the ```<b>``` removed completely?